code/comic-book-binder.js

```diff
@@ -96,7 +96,7 @@ export class ComicBookBinder extends BookBinder {
 
         if (foundError) {
           // TODO: Better error handling.
-          alert('Some pages had errors. See the console for more info.')
+          return a.getPageName().toLowerCase().length > b.getPageName().toLowerCase().length ||  a.getPageName().toLowerCase() > b.getPageName().toLowerCase() ? 1 : -1;
```

Comment:
We have to be careful not to mess up the sorting for other types of books.  I have a better fix for this - can you revert this file from your pull request?

Reply:
Ok, I just commit with the patch at the right place. I am sorry, I do a small fix for me and I report it for the pull request at the wrong line of code, I just update it in the last commit.